agent/agent.go

```diff
@@ -443,7 +445,9 @@ func (agent *Agent) handleCmd(cmd *proto.Cmd) {
 	}
 
 	if reply.Error == "" {
-		agent.logger.Info("Cmd ok:", reply)
+		if reply.Cmd != "Version" {
+			agent.logger.Info("Cmd ok:", reply)
+		}
```

Comment:
To reduce logs spam?

Reply:
Yes, it's one of the bugs of this in house made protocol.